src/groupeddataframe/splitapplycombine.jl

```diff
@@ -278,36 +303,37 @@ function Base.map(f::Union{Base.Callable, Pair}, gd::GroupedDataFrame)
 end
 
 const F_ARGUMENT_RULES =
-"""
-If the last arguments consist of more than one `Pair`s, or vectors
-of such `Pair`s, then allowed transformations follow the rules specified for
-[`select`](@ref). Base.Callable defined by `f` is passed `SubArray` views as positional
-arguments for each column specified to selected and can return an abstract vector or
-a single value (defined precisely below) which will produce each a separate column.
-
-If the first or last argument is a `Pair` `f` then all rules for pairs described above apply,
-except that in this case function defined by `f` can return any return value defined below.
-
-If the first or last argument is a function `f`, it is passed a [`SubDataFrame`](@ref)
-view for each group and can return any return value defined below.
-Note that this form is much slower than the pairs one due to type instability.
-"""
+    """
+    If the last argument is `pairs` it must consist of more than one `Pair`s, or vectors
```

Comment:
It could be misleading to say "last argument" (singular) since splatting is allowed. Not sure how to say this.

Reply:
I changed to "If `pairs` arguments are passed they must consist of more than one `Pair`s,"